core/ledger/ledger.go

```diff
@@ -0,0 +1,139 @@
+package ledger
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/sha256"
+	"os"
+	"time"
+
+	ledgerPb "github.com/TRON-US/go-btfs/core/ledger/pb"
+
+	ethereum "github.com/ethereum/go-ethereum/crypto"
```

Comment:
There's another implementation https://godoc.org/github.com/bitherhq/go-bither/crypto.
Do we want use ethereum or other alternative?

Reply:
@jin-tron will investigate into using https://github.com/libp2p/go-libp2p-core/blob/master/crypto/ecdsa.go directly since it's already a dependency